lib/commands/deployproxy.js

```diff
@@ -606,15 +606,19 @@ function uploadProxies(opts, request, done) {
           if (opts.verbose) {
             console.error('Deployment result: %j', body);
           }
-          var jsonBody = JSON.parse(body);
-          var errMsg;
-          if (jsonBody && (jsonBody.message)) {
-            errMsg = jsonBody.message;
-          } else {
-            errMsg = util.format('Error uploading proxy: %d',
-              req.statusCode);
+          try {
+            var jsonBody = JSON.parse(body);
```

Comment:
Can we avoid variable hoisting by putting the `var` statements outside of the `try`?  Refactor your code to handle this and we should be good.